llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -10160,7 +10160,7 @@ preparePlanForEpilogueVectorLoop(VPlan &Plan, Loop *L,
     if (!ExpandR)
       continue;
     auto *ExpandedVal =
```

Comment:
```suggestion
    VPValue *ExpandedVal =
```

now we can spell out the simple type